feature/search/presentation/src/main/java/com/sanaa/presentation/screen/componants/FilterSection.kt

```diff
@@ -0,0 +1,51 @@
+package com.sanaa.presentation.screen.componants
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sanaa.designsystem.R
+import com.sanaa.designsystem.design_system.theme.Theme
+
+@Composable
+fun FilterSection(
+    onClick: ()->Unit = {}
+) {
+    Box(
+        modifier = Modifier
+            .size(height = 48.dp, width = 52.dp)
```

Comment:
The values here not correct:
```suggestion
            .size(height = 52.dp, width = 48.dp)
```